src/app/zap-templates/zcl/air-pressure-measurement.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>Measurement &amp; Sensing</domain>
+    <name>Air Pressure Measurement</name>
+    <code>0x0407</code>
```

Comment:
This is the ZCL cluster id for the "Leaf Wetness" cluster.  This needs a different ID.  Please speak to @CamWms about one.

Reply:
I'm in talks with @CamWms regarding this issue. Will update it when I've got the correct cluster id.